### PR TITLE
Bump rubygem-openscap to 0.4.9-1

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: packages/plugins/rubygem-openscap/rubygem-openscap.spec [hunk 1]
```diff
@@ -1,66 +1,40 @@
...
 URL: https://github.com/OpenSCAP/ruby-openscap
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
+# start specfile generated dependencies
+Requires: ruby(release)
+Requires: ruby
 Requires: ruby(rubygems)
 Requires: rubygem(ffi) >= 1.0.9
-# require libopenscap.so.8 in an arch neutral way
-Requires: openscap >= 1.2.9
```
Comment (ekohl) [inline packages/plugins/rubygem-openscap/rubygem-openscap.spec]: I suspect you need to add this below the generated dependencies to make sure it's present.

File: packages/plugins/rubygem-openscap/rubygem-openscap.spec [hunk 2]
```diff
@@ -1,103 +1,92 @@
...
 
-Name: rubygem-%{gem_name}
-Version: 0.4.7
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.4.9
 Release: 1%{?dist}
 Summary: A FFI wrapper around the OpenSCAP library
 Group: Development/Languages
-License: GPLv2+
+License: GPL-2.0
```
Comment (ekohl) [inline packages/plugins/rubygem-openscap/rubygem-openscap.spec]: I think this should be GPLv2 or GPLv2+ so it matches the Fedora license codes.

**Comment by ehelms** [comment]
Updated

**Comment by evgeni** [comment]
@ekohl we never shipped the `-devel` package in our comps, so while you're *technically* correct -- nobody should *need* the obsoletes.

That said, we also don't have `-doc` in comps :)

**Pull request closed without merging**